pkg/kubectl/cmd/get/humanreadable_flags_test.go

```diff
@@ -44,6 +44,7 @@ func TestHumanReadablePrinterSupportsExpectedOptions(t *testing.T) {
 		// TODO(juanvallejo): test sorting once it's moved to the HumanReadablePrinter
 		sortBy       string
 		columnLabels []string
+		extraColumns []string
```

Comment:
Since you're already in this file, consider adding a new test case here for extraColumns

Reply:
Also, add a test case that tries adding an extraColumn with an "uncommon" value (intead of .metadata.name, for example, have the value be a .spec field)

Reply:
Still working on adding some tests for these changes. Should have something in by tonight hopefully.